src/briefcase/commands/build.py

```diff
@@ -14,16 +17,66 @@ def add_options(self, parser):
         self._add_update_options(parser, context_label=" before building")
         self._add_test_options(parser, context_label="Build")
 
-    def build_app(self, app: BaseConfig, **options):
+    def build_app(self, app: AppConfig, **options):
         """Build an application.
 
         :param app: The application to build
         """
         # Default implementation; nothing to build.
 
+    def verify_template(self, app: AppConfig):
```

Comment:
This should probably be in the base command, and called as part of a more generic check (`finalize?`). If you've got an app that was generated for 0.3.14, and you try to run it without building, you're going to have the same class of problem - it's not exclusively a *build* problem.

Reply:
Yeah...I initially tried to put it there but ran in to issue. I'll try to move it up again.